DataFormats/Reconstruction/include/ReconstructionDataFormats/Vertex.h

```diff
@@ -143,8 +143,8 @@ class Vertex : public VertexBase
   GPUd() void setTimeStamp(const Stamp& v) { mTimeStamp = v; }
 
  protected:
-  float mChi2 = 0;           ///< chi2 or quality of tracks to vertex attachment
   ushort mNContributors = 0; ///< N contributors
+  float mChi2 = 0;           ///< chi2 or quality of tracks to vertex attachment
```

Comment:
Not sure we need to respect this stupid `-Wreorder` warning, I think since even c+11 the ordering is not required.
In any case, please don't change the class layout (moreovere from 4B,2B,2B to 2B,4B,2B) but swap the order of initialization in the https://github.com/AliceO2Group/AliceO2/blob/cdb68743ea3d7122f7eb25d286a9032d0e34e6ae/DataFormats/Reconstruction/include/ReconstructionDataFormats/Vertex.h#L125

Reply:
Ups, I completely forgot about padding! sure I'll do that.